### DISCUSSION TITLE
無法在github pages上顯示

### DISCUSSION THREAD

**Discussion by nataliepjlin** [ppoffice/hexo-theme-icarus#1055] [Q&A]
repo: https://github.com/nataliepjlin/icarus.git

在local4000上運行順利，但github pages上是空白的。
____________________________________________________
it works perfectly fine on local4000, but on github pages, it's blank! Does anyone know how to fix it?

**Comment by YiSunYiMao** [comment]
将根_config.yml文件最后 branch: master改为main试一下

**Reply by YiSunYiMao** [reply]
或者看看我写的两个专栏https://space.bilibili.com/281523005/article

**Reply by nataliepjlin** [reply]
@YiSunYiMao 更新了main或重新生成了ssh key還是不行呢 查看`git remote -v`也是正確的

**Comment by YiSunYiMao** [comment]
参考一下，再试一试，是否是仓库格式不太标准，根_config.yml文件的最后
deploy:
  type: git
  repository: git@github.com:YiSunYiMao/YiSunYiMao.github.io.git
  branch: main

**Reply by nataliepjlin** [reply]
@YiSunYiMao 已解決，是url出問題了。感謝
